metrics/examples/prometheus/install/01-prometheus-operator.yaml

```diff
@@ -0,0 +1,147 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I guess that you copied this one from somewhere. I think that we should not have Prometheus operator installation files here because they could change upstream somehow and we should have them insync. Can we reference the installation process in the doc? Or did you change the YAML file for having it working on OpenShift for any reason?